### PR TITLE
Document helpers

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

**Comment by pvenkatakrishnan** [comment]
isnt that an array ?

File: README.md [hunk 1]
```diff
@@ -437,7 +437,10 @@ kraken-js looks to the `view engines` config property to understand how to load
             "module": "adaro",
             "renderer": {
                 "method": "dust",
-                "arguments": [{ "cache": false }]
+                "arguments": [{
+                    "cache": false,
+                    "helpers": "dust-helpers-whatevermodule"
+                }]
```
Comment (pvenkatakrishnan) [inline README.md:443]: helpers is an array i thought

Comment (aredridel) [inline README.md:443]: OH derp. You are right.

Comment (aredridel) [inline README.md:443]: Fixed in next rev.

**Comment by pvenkatakrishnan** [comment]
:+1:

**Comment by totherik** [comment]
:+1:

**Pull request merged**